apps/api/src/queries/buildViewModelForResource/viewModels/base-resource.view-model.ts

```diff
@@ -27,7 +28,7 @@ export class BaseResourceViewModel extends BaseViewModel {
             .getContributions()
             .map(({ contributorId }) => contributors.find(idEquals(contributorId)))
             .filter((contributor) => !isNullOrUndefined(contributor))
-            .map((contributor) => contributor.fullName.toString());
+            .map(({ id, fullName }) => ({ id: id, fullName: fullName.toString() }));
```

Comment:
I know this is my design, but just reminds me that this is very slow if we have more than a few contributors. We need to move to the denormalized materialized views soon.